### ISSUE TITLE
[Documentation] Can not initialize the QueueMessagingTemplate

### ISSUE THREAD

**Issue by larchanjo** [spring-cloud/spring-cloud-aws#537]
There is wrong [documentation](https://cloud.spring.io/spring-cloud-aws/reference/html/#sending-a-message) about Amazon Simple Queue Service (AWS SQS) related to [QueueMessagingTemplate](https://www.javadoc.io/static/org.springframework.cloud/spring-cloud-aws-messaging/2.2.0.RELEASE/org/springframework/cloud/aws/messaging/core/QueueMessagingTemplate.html)

```
import com.amazonaws.services.sqs.AmazonSQS;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.cloud.aws.messaging.core.QueueMessagingTemplate;
import org.springframework.messaging.support.MessageBuilder;

public class SqsQueueSender {

    private final QueueMessagingTemplate queueMessagingTemplate;

    @Autowired
    public SqsQueueSender(AmazonSQS amazonSqs) {
        this.queueMessagingTemplate = new QueueMessagingTemplate(amazonSqs);
    }

    public void send(String message) {
        this.queueMessagingTemplate.send("physicalQueueName", MessageBuilder.withPayload(message).build());
    }
}
```

Is not possible to initialize [QueueMessagingTemplate](https://www.javadoc.io/static/org.springframework.cloud/spring-cloud-aws-messaging/2.2.0.RELEASE/org/springframework/cloud/aws/messaging/core/QueueMessagingTemplate.html) using the **AmazonSQS**

It is only possible, using the **AmazonSQSAsync**

```
import com.amazonaws.services.sqs.AmazonSQSAsync;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.cloud.aws.messaging.core.QueueMessagingTemplate;
import org.springframework.messaging.support.MessageBuilder;

public class SqsQueueSender {

    private final QueueMessagingTemplate queueMessagingTemplate;

    @Autowired
    public SqsQueueSender(AmazonSQSAsync amazonSQSAsync) {
        this.queueMessagingTemplate = new QueueMessagingTemplate(amazonSQSAsync);
    }

    public void send(String message) {
        this.queueMessagingTemplate.send("physicalQueueName", MessageBuilder.withPayload(message).build());
    }
}
```

**Comment by maciejwalkowiak** [comment]
Thanks @larchanjo for reporting. Would you like to contribute a fix to documentation?

**Comment by larchanjo** [comment]
Hi, @maciejwalkowiak I would like, but I don't know where is the documentation repo, can you help me?

**Comment by maciejwalkowiak** [comment]
Sure! It's in this very github repository: https://github.com/spring-cloud/spring-cloud-aws/blob/master/docs/src/main/asciidoc/spring-cloud-aws.adoc

**Comment by larchanjo** [comment]
@maciejwalkowiak Thank you soo much, the PR is #558

**Comment by maciejwalkowiak** [comment]
Thanks @larchanjo. Merged in #558 😄
